jdaviz/core/helpers.py

```diff
@@ -35,3 +35,49 @@ def __init__(self, app=None):
 
     def load_data(self, data, parser_reference=None, **kwargs):
         self.app.load_data(data, parser_reference=parser_reference, **kwargs)
+
+    @property
+    def fitted_models(self):
+        """
+        Returns the fitted model parameters.
+
+        Returns
+        -------
+        parameters : dict
+            dict of Quantity arrays, or None.
+        """
+        all_models = {}
+        if hasattr(self.app, '_fitted_1d_models'):
+            all_models = {**all_models, **self.app._fitted_1d_models}
+        if hasattr(self.app, '_fitted_3d_model'):
+            all_models = {**all_models, **self.app._fitted_3d_model}
+
+        return all_models
+
+    def get_1d_models(self):
```

Comment:
Interpreting what @eteq said this morning, I was thinking more like:

```python
def get_models(ndim=1):
    attrname = f'_fitted_{ndim}d_models'
    return getattr(self, attrname, None)
```

Reply:
Ah, ok, I think that makes more sense!

Reply:
p.s. `fitted_models` probably can call `get_models` somehow to avoid code duplication.

Reply:
Thank you for the suggestion!